src/algorithms/calorimetry/ImagingClusterReco.h

```diff
@@ -252,8 +243,10 @@ class ImagingClusterReco {
         // shower radius estimate (eta-phi plane)
         double radius = 0.;
         for (const auto &hit: hits) {
-            radius += pow2(edm4eic::eta(hit.getPosition()) - edm4eic::eta(cluster.getPosition())) +
-                      pow2(edm4eic::angleAzimuthal(hit.getPosition()) - edm4eic::angleAzimuthal(cluster.getPosition()));
+            radius += std::hypot(
+                (edm4eic::eta(hit.getPosition()) - edm4eic::eta(cluster.getPosition())),
+                (edm4eic::angleAzimuthal(hit.getPosition()) - edm4eic::angleAzimuthal(cluster.getPosition()))
+            );
         }
         cluster.addToShapeParameters(std::sqrt(radius / cluster.getNhits()));
```

Comment:
The dimensions of the parameter has changed here: radius here is not L^2, but L. If this is intentional, I'd submit and discuss separately.

Reply:
Good catch. Unintentional.

Reply:
The cluster shape radius is supposed to be of dimension [mm] according to the data model. The sqrt was missing before. After the ImagingClusterReco has been merged here, we can fix the algorithm itself to remove the pow again.